Source/items.cpp

```diff
@@ -2304,6 +2304,8 @@ void SaveItemPower(int i, item_effect_type power, int param1, int param2, int mi
 		items[i]._iPLHP -= (r2 << 6);
 		items[i]._iPLMana += (r2 << 6);
 		break;
+	default:
+		break;

```

Comment:
I'd rather fail fast here:
```suggestion
		app_fatal("Invalid item power");
```

Reply:
there are multiple powers that do not get applied by this switch

Reply:
Ah ok.